Makefile directive undefine not available on anlworkstation

The gmake utility (GNU Make 3.81) available on compute001 does not
support the undefine directive. A build on compute001 fails with the following
error message,
```
/sandbox/jayesh/ESMCI_cime/scripts/test_mpi_serial4/Tools/Makefile:228: *** missing separator.  Stop.
if ( 0 == 1 ) then
cp -p Makefile.conf Makefile.conf.old
cp: cannot stat ‘Makefile.conf’: No such file or directory
make SRCDIR=/sandbox/jayesh/ESMCI_cime/externals/mct
Makefile:3: Makefile.conf: No such file or directory
make: *** No rule to make target `Makefile.conf'.  Stop.
exit 1
```
The undefine directive is supported on newer versions of gmake.
The undefine directive was added to cime_config/acme_machines/Makefile in 3dbe9591951f63b17469a2a00f6dbbe1f09dc302

--- a/cime_config/acme/machines/Makefile
+++ b/cime_config/acme/machines/Makefile
@@ -225,7 +225,9 @@ endif
 
 
 ifeq ($(MPILIB),mpi-serial)
-  undefine PNETCDF_PATH
+	ifdef PNETCDF_PATH
+		undefine PNETCDF_PATH
+	endif
 else
   ifdef PNETCDF_PATH
     ifndef $(INC_PNETCDF)

